src/main/java/org/folio/search/integration/KafkaMessageListener.java

```diff
@@ -153,6 +157,28 @@ public void handleConsortiumInstanceEvents(List<ConsumerRecord<String, Consortiu
     }));
   }
 
+  @KafkaListener(
+    id = KafkaConstants.CLASSIFICATION_TYPE_LISTENER_ID,
+    containerFactory = "standardListenerContainerFactory",
+    groupId = "#{folioKafkaProperties.listener['classification-type'].groupId}",
+    concurrency = "#{folioKafkaProperties.listener['classification-type'].concurrency}",
+    topicPattern = "#{folioKafkaProperties.listener['classification-type'].topicPattern}")
+  public void handleClassificationTypeEvents(List<ConsumerRecord<String, ResourceEvent>> consumerRecords) {
+    log.info("Processing classification-type events from Kafka [number of events: {}]", consumerRecords.size());
+    var batch = consumerRecords.stream()
+      .map(ConsumerRecord::value)
+      .filter(resourceEvent -> resourceEvent.getType() == DELETE).toList();
```

Comment:
maybe makes sense to move this filtration to ```ResourceChangeFilterStrategy```

Reply:
When a record passed into `ResourceChangeFilterStrategy` the resourceName field of the payload is still empty because in mod-inventory-storage it's not populated.

Reply:
Ok, got it.